sdk/identity/msal.yml

```diff
@@ -0,0 +1,36 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+trigger:
+  branches:
+    include:
+    - master
+    - hotfix/*
+    - release/*
+    - restapi*
+  paths:
+    include:
+    - sdk/identity/
+    - sdk/core/
+
+pr:
```

Comment:
Does this mean the pipeline will run on PRs?

Reply:
yes. I will correct this to make it only scheduled run. thanks